packages/components-css/card-group/index.scss

```diff
@@ -0,0 +1,28 @@
+.frameless-card-group {
+  container-type: inline-size;
+  display: grid;
+  gap: 24px;
+  grid-auto-flow: dense;
+}
+
+@media (width >= 600px) {
+  .frameless-card-group {
+    display: grid;
+    gap: 24px;
```

Comment:
Deze `display` en `gap` lijken me niet nodig als hij hetzelfde is als de parent?